Source/Core/Core/FifoPlayer/FifoDataFile.h

```diff
@@ -51,6 +51,7 @@ class FifoDataFile
     CP_MEM_SIZE = 256,
     XF_MEM_SIZE = 4096,
     XF_REGS_SIZE = 96,
+    TEX_MEM_SIZE = 1024 * 1024,
```

Comment:
Uh, it's the same style as the old code, but I feel bad not fetching this information from videocommon :/

Maybe using vectors instead, but this is out of scope of this PR.